pymc_marketing/mlflow.py

```diff
@@ -548,12 +965,15 @@ def new_sample(*args, **kwargs):
                     **arviz_summary_kwargs,
                 )
 
-            model = pm.modelcontext(kwargs.get("model"))
             if log_model_info:
                 log_model_derived_info(model)
 
-            if log_datasets:
-                log_data(model=model, idata=idata)
+            if log_metadata_info:
+                log_metadata(model=model, idata=idata)
+
+            # mmm.fit() calls pm.sample() internally, so we need to make sure log-likelihood is only added once
+            if log_loocv and "log_likelihood" not in idata.groups():
+                pm.compute_log_likelihood(idata=idata, model=model)
```

Comment:
Dont automatically call with sample. It can be high computation

Reply:
But isn't this taken care of by `log_loocv=False` as default for the arg? Like the computation only happens if the user specifies they want loocv metrics?

Reply:
In my mind, the automatic logging should be triggered with pm.compute_log_likelihood call or equivalent method wrapper

Reply:
Hmm, isn't this a bit confusing though? Like the creation of LOOCV metrics is handled by `az.loo` - I would find it surprising that `pm.compute_log_likelihood` does this automatically, since usually people would have to do this manually.

I suppose I could wrap `az.loo` instead, but I'm not sure that's simpler since people would still need to know to do `pm.compute_log_likelihood` first. 

What do you think about just updating the doc-string to warn people that if `log_loocv=True` then  `pm.compute_log_likelihood` is called, which can be high computation? @wd60622

Reply:
I think wrapping az.loo makes more sense  then. Since we don't have that functionality in ModelBuilder yet, let's hold off on this implementation so this PR doesn't become more than it already is